SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -743,7 +752,9 @@ int64 USpatialActorChannel::ReplicateActor()
 
 	bForceCompareProperties = false;		// Only do this once per frame when set
 
-	return (bWroteSomethingImportant) ? 1 : 0;	// TODO: return number of bits written (UNR-664)
+	INC_DWORD_STAT_BY(STAT_SpatialNumReplicatedActorBytes, ReplicationBytesWritten);
```

Comment:
No need to create our own variable - lets just use `STAT_NumReplicatedActorBytes`